controllers/clustergroupupgrade_controller.go

```diff
@@ -335,6 +336,9 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 			return
 		}
 
+		if clusterGroupUpgrade.Status.Status.StartedAt.IsZero() {
+			clusterGroupUpgrade.Status.Status.StartedAt = metav1.Now()
+		}
```

Comment:
won't this cause an issue when the cgu starts later, since this will be used to check for cgu timeout?
I would instead expect the backup started at field you added to be initialized somewhere

Reply:
It's initialized here:
```
	if clusterGroupUpgrade.Spec.Backup {
		// Backup is required
		if clusterGroupUpgrade.Status.Backup == nil {
			clusterGroupUpgrade.Status.Backup = &ranv1alpha1.BackupStatus{
				Status:    make(map[string]string),
				Clusters:  []string{},
				StartedAt: metav1.Now(),
			}
		}
```

Reply:
Not sure what you mean by later. CGU starts before backup starts.